features/host_can_make_offer_on_listing.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Host can make an offer on a listing
+  As a host,
+  In order to make some money,
+  I want to be able to make an offer on a listing
+
+  Background:
+    Given the following listings exist
+    | pet_name  | pet_location    | pet_description   | start_date      | end_date      | pet_picture |
+    | Zane      | Gothenburg      | I'm nice          | 2019-06-28      | 2019-06-29    | picture1    |
+    | Carla     | Stockholm       | I love cats!      | 2019-05-28      | 2019-06-01    | picture2    |
+
+    When I visit the landing page
+    Then I should see "Zane"
+    And I should see "Gothenburg" within "Zane" section
+    And I should see "I'm nice" within "Zane" section
+    And I should see "2019-06-28" within "Zane" section
+    And I should see "2019-06-29" within "Zane" section
+    And I should see "picture1" within "Zane" section
+
+  Scenario: Host can successfully create an offer
+    When I click "Make an offer" within "Zane" section
+    Then I should be on the "Zane" listing page
+    Then I should see "Create your offer"
```

Comment:
So you press "Make an offer" to be able to press another button which says "Create your offer". Am I correct that the "Make an offer" leads to the listing show page?

Reply:
I would suggest that you redirect the "Make an offer" button from the listing show page to the "new" offer page.

Reply:
We had this discussion with Faraz, and we come to an agreement that we would keep it in the show view

Reply:
this time around